packages/@aws-cdk/aws-synthetics/test/canary.test.ts

```diff
@@ -304,7 +304,7 @@ test('Schedule can be set with Cron', () => {
       handler: 'index.handler',
       code: synthetics.Code.fromInline('/* Synthetics handler code */'),
     }),
-    runtime: synthetics.Runtime.SYNTHETICS_NODEJS_PUPPETEER_3_3,
+    runtime: synthetics.Runtime.SYNTHETICS_NODEJS_PUPPETEER_3_4,
```

Comment:
I'd rather not change this. Since we're not testing for runtime, it doesn't matter what we change in this field.

Reply:
Reverted in 24855a933.